lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
I get your point, it could be reused just like `minuteInterval`, for instance. I want to clarify that I don't have a strong opinion in favor of one thing or another, just that I feel that it configures the template the same way that the `DatePickerIOS` props.

Regarding your example, you could actually reuse the `background` prop in order to make all the touchable components behave the same.